src/gameComponents/Arrow.vue

```diff
@@ -16,9 +17,17 @@ import { EActionArrow, EGetterArrow, IArrowData } from '@/store/modules/arrow/ty
 @Component
 export default class Arrow extends Vue {
   arrowData!: IArrowData
+  iconName!: string
+  iconsMap: {[index: string]: string} = {
+    'left-arrow': 'arrow-left-square',
+    'up-arrow': 'arrow-up-square',
+    'down-arrow': 'arrow-down-square',
+    'right-arrow': 'arrow-right-square'
+  }
 
   created () {
     this.arrowData = this.getLastArrow()
```

Comment:
Can't understand why it's necessary to get last arrow in each Arrow component (not for the last one, but for each)

Reply:
@silchencko by pressing `start` button in the store create an array with arrow data **objects**(id and dirrection/className). After that, if there is an array with data for the arrows, they begin to be rendered. But before rendering, each `arrow component` takes the data of the last element of the array in order to assign itself with id and a direction (class). This is necessary because the arrows need need to be different and placed in the right places.
Also i need to get arrow class in `arrow component` because all this styles initialised in this component for avoiding extra styles for the arrow in the `border component`


I renamed **getLastArrow()** to **getLastArrowFromStore()** for better understanding. 

If it's not clear, I can try to explain in more detailed.